koku/api/report/test/ocp/test_ocp_query_handler.py

```diff
@@ -321,9 +310,12 @@ def test_add_current_month_deltas_no_previous_data_wo_query_data(self):
 
             result = handler.add_current_month_deltas(query_data, query_sum)
 
+            expected_delta = query_sum.get('usage', 0) - query_sum.get('request', 0)
+            expected_percent = query_sum.get('usage', 0) / query_sum.get('request', 0) * 100
+
             self.assertEqual(result, query_data)
-            self.assertEqual(handler.query_delta['value'], Decimal(0))
-            self.assertIsNone(handler.query_delta['percent'])
+            self.assertEqual(handler.query_delta['value'], expected_delta)
+            self.assertEqual(handler.query_delta['percent'], expected_percent)
```

Comment:
This test change was required because the delta between `usage` and `request` was non-zero. I'm not sure what changed in the data generator to cause the values to differ, but it means the test wasn't testing the right thing - the expected values would never be zero, as far as I could tell.